contrib/fuzz/unstable_call_graph.py

```diff
@@ -0,0 +1,227 @@
+        if node.get('is_unstable', False):
+            return True
+        if 'children' in node:
+            for child in node['children'].values():
+                if self.has_flag_true(child):
+                    return True
+        return False
+
+
+def demangle(symbol):
+    try:
+        if symbol.endswith('@plt'):
+            symbol = symbol[:-4]
+        result = subprocess.run(['c++filt', symbol], capture_output=True, text=True, check=True)
+        return result.stdout.strip()
+    except subprocess.CalledProcessError:
+        return symbol
+
+
+def get_architecture(file_path):
+    try:
+        result = subprocess.run(['file', file_path], capture_output=True, text=True)
+        output = result.stdout.strip()
+
+        if 'x86-64' in output:
+            return 'x86_64'
+        elif 'aarch64' in output or 'arm64' in output:
+            return 'arm64'
+        else:
+            return 'Unknown'
+
+    except FileNotFoundError:
+        print("The 'file' command is not available. Ensure it's installed.")
+        return None
+    except Exception as e:
+        print(f"An error occurred: {e}")
+        return None
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description='''Creates a function hierarchy and highlights unstable functions''',
+    )
+    parser.add_argument(
+        "--afl-edge-id-file",
+        help="Path to afl++'s file containing all edge ids and symbols of the binary",
+        type=str,
+        required=True)
+    parser.add_argument(
+        "--afl-fuzzer-stats",
+        help="Path to afl++'s fuzzer_stats file",
+        type=str,
+        required=True)
+    parser.add_argument(
+        "--binary-file",
+        help="Path to binary file from which to generate the call graph",
+        type=str,
+        required=True)
+    parser.add_argument(
+        "--target-function",
+        help="The target function that is the top of the call graph (e.g. fuzz_target_name_fuzz_target)",
+        type=str,
+        required=True)
+    parser.add_argument(
+        "--ignored-prefixes",
+        help="Path to file that contains strings that if at the start of a function name, that function is skipped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--dump-hierarchy",
+        help="Path to json file where the hierarchy will be dumped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--dump-unstable",
+        help="Path to file where unstable symbols will be dumped",
+        type=str,
+        default=None)
+    parser.add_argument(
+        "--no-tui",
+        help="Whether to output the hierarchy to the terminal",
+        action="store_true")
+
+    args = parser.parse_args()
+
+    unstable_edges = []
+    unstable_functions = set()
+    prefixes_to_ignore = set()
+
+    with open(args.afl_fuzzer_stats, 'r') as fuzzer_stats_file:
```

Comment:
As discussed yesterday, some log messages indicating that the script is making progress would be nice.

e.g. "Read unstable edge ids from fuzzer_stats", "Dumped unstable edge symbols to \<file\>", ...